client/deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/template/OpenApiNamespaceResolver.java

```diff
@@ -58,6 +64,26 @@ public boolean hasAuthMethods(OperationMap operations) {
         return operations != null && operations.getOperation().stream().anyMatch(operation -> operation.hasAuthMethods);
     }
 
+    /**
+     * Ignore the OAuth flows by filtering every oauth instance by name. The inner openapi-generator library duplicates the
+     * OAuth instances per flow in the openapi spec.
+     * So a specification file with more than one flow defined has two entries in the list. For now, we do not use this
+     * information in runtime so it can be safely filtered and ignored.
+     *
+     * @param oauthOperations passed through the Qute template
+     * @see "resources/templates/libraries/microprofile/auth/compositeAuthenticationProvider.qute"
+     * @return The list filtered by unique auth name
+     */
+    public List<CodegenSecurity> getUniqueOAuthOperations(ArrayList<CodegenSecurity> oauthOperations) {
+        if (oauthOperations != null) {
+            return oauthOperations.stream()
+                    .collect(Collectors.toMap(security -> security.name, security -> security,
+                            (existing, replacement) -> existing, LinkedHashMap::new))
+                    .values().stream().toList();
+        }
+        return Collections.emptyList();
```

Comment:
```suggestion
        return oauthOperations != null  ? 
          new ArrayList<>(oauthOperations.stream()
                    .collect(Collectors.toMap(security -> security.name, security -> security,
                            (existing, replacement) -> existing, LinkedHashMap::new))
                    .values()) : Collections.emptyList();
```

Reply:
I don't do oneliners, but I'll change to the `ArrayList`constructor.